### ISSUE TITLE
Not usable in combination with wired LED

### ISSUE THREAD

**Issue by JesseFPV** [Kian-Wee/rh_wled_handler_strip#1]
Hi.

Nice project, unfortunately its not working in combination with wired led because you have to replace the LED actios with WLED actions. 

As we use a wired led strip on the start gate and planning to use WLED on other gates it may need another approach.

**Comment by Kian-Wee** [comment]
Yeah, initially I wanted to have WLED mirror the existing local LED effects. But after speaking to Hazard, it seemed like the recommended implementation would be to have it as a separate plugin from the rh_led_handler_strip and register WLED as separate effects. So unless you want to fork the main rh code and merge wled changes to the rh_led_handler_strip every update, Im not sure how this could be done. I ended up just running WLED for all the gates.

**Comment by Kian-Wee** [comment]
One other thing that I noticed during development is that its quite hard to get the exact same effects with the rh LED library and the WLED effects library. I tried finding the most similar WLED effects but often times even the colors were of different shades.

**Comment by JesseFPV** [comment]
Thanks for your feedback! Would it possible to, when WLED effect is selected, it will also uses the standard effect? So it would include both?

**Comment by Kian-Wee** [comment]
Unfortunately with the LED events tab is also handled by rh so I cant think of a way to implement this functionality without modifying the base code, I can only register WLED effects as a new effect, not an extension of an existing one.
